### PR TITLE
howto/warning-memory-issues

### PR DESCRIPTION
**Pull request by ecuriotto**
After using the office connector with different customers, and experiencing memory issues in the Bonita Studio, I think it's better to warn the users about this issues, and propose a solution.

### CODE REVIEW

**Review by alachambre — APPROVED** [review]

File: md/generate-pdf-from-an-office-document.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ The generated document can then be sent by email or stored in an external inform
 For proper conversion, you must specify the character encoding that is used in the source document. On most Windows configurations, encoding **"windows-1252"** is used.  
 UTF-8 is assumed if you do not specify the encoding (that is, if the field is left blank in the connector configuration wizard).   
 
+**Warning:** This connector requires memory for its dependencies, if you experience issues you can increase the memory in your Bonita Studio: Preferences-> Server settings -> 1024 (0r 2048)
```
Comment (educhastenier) [inline md/generate-pdf-from-an-office-document.md:14]:
```suggestion
**Warning:** This connector requires much memory for its dependencies, if you experience issues you can increase the memory in your Bonita Studio: Preferences-> Server settings -> 1024 (or 2048)
```

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

**Comment by tbouffard** [comment]
@ecuriotto @educhastenier do we integrate the last suggestion or do we merge as it is ?

**Comment by rbioteau** [comment]
Closing as it is covered by https://github.com/bonitasoft/bonita-doc/pull/1075

**Pull request closed without merging**